google-cloud-error_reporting/lib/google/cloud/error_reporting/rails.rb

```diff
@@ -64,9 +64,7 @@ class Railtie < ::Rails::Railtie
             service_version = er_config.service_version
 
             error_reporting = V1beta1::ReportErrorsServiceClient.new(
-              channel: channel,
-              app_name: service_name,
-              app_version: service_version
+              channel: channel
```

Comment:
The lib_name and lib_version specify the library that is wrapping gapic. The prior was passing through the service name and version from the user not specifying the library that is wrapping the gapic lib_name and lib_version are used for.